packages/quantic/force-app/main/default/lwc/quanticPager/e2e/fixture.ts

```diff
@@ -0,0 +1,63 @@
+import {PagerObject} from './pagerObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+
+const pagerUrl = 's/quantic-pager';
+
+interface PagerOptions {
+  numberOfPages: number;
+}
+type QuanticPagerE2EFixtures = {
+  pager: PagerObject;
+  search: SearchObject;
+  options: Partial<PagerOptions>;
+};
+
+type QuanticPagerE2ESearchFixtures = QuanticPagerE2EFixtures & {
+  urlHash: string;
+};
+
+type QuanticPagerE2EInsightFixtures = QuanticPagerE2ESearchFixtures & {
+  insightSetup: InsightSetupObject;
+};
+
+export const testSearch = quanticBase.extend<QuanticPagerE2ESearchFixtures>({
+  options: {},
```

Comment:
Are these coming from the `test.use` ?

This is what I was thinking we could do to simplify the useCases too:

```
test.use({
    useCase: 'search|insight',
});

// And then in your fixture:
pager: async ({page, ..., useCase}, use) => {
    await use(new PagerObject(page, useCase))
}

// And then in your PagerObject, for example instead of having a param on 
async waitForPagerUaAnalytics() {
    if (this.useCase === 'insight') {...}
    if (this.useCase === 'search') {...}
}
```

Basically is there a way to not have two fixtures, `testInsight` & `testSearch` and simply have a `testPager` fixture that instead gets a `useCase` as an option?

Reply:
Currently the Pager Object is fully independent of  the use case, it doesn't care at all about which use case you are is, which is my sense ideal. I'm not understanding why we would want to make it take a use case as a parameter?